app/presenters/queries/content_item_presenter.rb

```diff
@@ -102,7 +102,9 @@ def select_fields(scope)
           when :base_path
             "content_items.base_path as base_path"
           when :locale
-            "content_items.locale as locale"
+            "documents_content_items.locale as locale"
```

Comment:
This was the name that Rails gave to the join of documents in this case, because there was a subquery with a join on the name `documents` already. I agree it's unclear though and I don't really like it, will try and find a way to make it clearer.